openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -138,6 +143,7 @@ def to_representation(self, user):  # lint-amnesty, pylint: disable=arguments-di
             "date_joined": user.date_joined.replace(microsecond=0),
             "last_login": user.last_login,
             "is_active": user.is_active,
+            "activation_key": None,
```

Comment:
Just wondering why this is set to `None`, Why this can't use the value of `activation_key` variable here? is there a reason to do this way?

Reply:
I was following the convention that was used by other variables in case the object does not exist but I believe I can use the variable directly here as you suggested.